src/app/model/User.old-test.ts

```diff
@@ -0,0 +1,242 @@
+// import ReduxFirebase from "react-redux-firebase";
```

Comment:
Just deprecating this test. I don't think anything it tests even exists any more